AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_CubeMapCaptureAdded.py

```diff
@@ -87,22 +108,39 @@ def AtomEditorComponents_CubeMapCapture_AddedToEntity():
         general.idle_wait_frames(1)
         Report.result(Tests.removal_undo, cubemap_capture_entity.has_component(AtomComponentProperties.cube_map_capture()))
 
-        # 5. Delete Cubemap_Capture entity.
+        # 5. Set Specular IBL property: Very High
+        cubemap_capture_component.set_component_property_value(AtomComponentProperties.cube_map_capture('Specular IBL'), value=SPECULAR_IBL_QUALITY['Very High'])
+        Report.result(Tests.specular_ibl_property_set_to_very_high, cubemap_capture_component.get_component_property_value(AtomComponentProperties.cube_map_capture('Specular IBL')) == SPECULAR_IBL_QUALITY['Very High'])
```

Comment:
for line length consideration you should new line indent in several places using the parenthesis
```suggestion
        cubemap_capture_component.set_component_property_value(
            AtomComponentProperties.cube_map_capture('Specular IBL'),
            value=SPECULAR_IBL_QUALITY['Very High'])
        Report.result(
            Tests.specular_ibl_property_set_to_very_high,
            cubemap_capture_component.get_component_property_value(
                AtomComponentProperties.cube_map_capture('Specular IBL')) == SPECULAR_IBL_QUALITY['Very High'])
```

Reply:
broke up for readability using parenthesis